src/components/Actions/Actions.css

```diff
@@ -69,8 +63,18 @@
     color: white;
   }
 
+  .operations {
+    flex-grow: 1;
+    justify-content: flex-end;
```

Comment:
Operation button will still be located in right of screen in iPad portrait mode.